graylog2-server/src/main/java/org/graylog2/alarmcallbacks/AlarmCallbackHistoryImpl.java

```diff
@@ -31,6 +32,7 @@
 import org.mongojack.ObjectId;
 
 @AutoValue
+@WithBeanGetter
```

Comment:
Why do we want to add the annotation to every AutoValue class we have? This only makes sense for those that have validation annotations, no?